shpkpr/commands/cmd_config.py

```diff
@@ -53,16 +52,15 @@ def set(ctx, application_id, env_vars):
 def unset(ctx, application_id, keys):
     """Unset application configuration.
     """
-    application = ctx.marathon_client.get_application(application_id)
+    existing_application = ctx.marathon_client.get_application(application_id)
+    application = MarathonApplication({'id': application_id})
```

Comment:
I'm inclined towards a no on that, imo the proper solution to that is just to make it possible to resubmit an application exactly as we get it back from `get_application` if we need that functionality.

I like the explicitness of setting only the fields to be changed, but I can see the merit of both approaches.